src/test/java/domain/ExpelStatusTest.java

```diff
@@ -0,0 +1,46 @@
+package domain;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+public class ExpelStatusTest {
+
+    @Test
+    void 경고_상태_확인() {
+        Map<AttendanceStatus, Integer> attendanceStatuses = new HashMap<>();
+        attendanceStatuses.put(AttendanceStatus.ATTEND, 3);
+        attendanceStatuses.put(AttendanceStatus.LATE, 3);
+        attendanceStatuses.put(AttendanceStatus.ABSENT, 1);
+        attendanceStatuses.put(AttendanceStatus.UNATTEND, 0);
```

Comment:
```suggestion
        Map<AttendanceStatus, Integer> attendanceStatuses = Map.of(
                AttendanceStatus.ATTEND, 3,
                AttendanceStatus.LATE, 3,
                AttendanceStatus.ABSENT, 1,
                AttendanceStatus.UNATTEND, 0
        );
```

위와 같이 Java 컬렉션에서 제공하는 메서드를 활용해보면 어떨까요?
리버의 방식과 어떤 차이가 있는지도 학습해보면 좋을 듯 해요.
더 나아가 Java 버전에 따라 어떤 메서드들이 있는지도 보면 좋구요 :)